punchcut/src/scaler.rs

```diff
@@ -0,0 +1,172 @@
+use super::{source::*, Context, Error, Hinting, NormalizedCoord, PathSink, Result, Variation};
```

Comment:
I thought import * was bad practice in Rust?

Reply:
It's really not necessary here as the only item in there is `glyf`. I'll fix it.

On the broader subject, I think it's generally fine for *private* imports when you're dealing with large sets of symbols. It's also common when libraries offer `prelude` modules which are designed to be glob imported.